lib/content-services/src/lib/search/components/search-filter-chips/search-filter-chips.component.html

```diff
@@ -3,6 +3,12 @@
         <adf-search-widget-chip [category]="category"></adf-search-widget-chip>
     </ng-container>
 
+    <ng-container *ngIf="facetFiltersService.tabbedFacet && showContextFacets">
+        <adf-search-facet-chip-tabbed
+            [tabbedFacet]="facetFiltersService.tabbedFacet"
+            [attr.data-automation-id]="'search-fact-chip-tabbed' + facetFiltersService.tabbedFacet.fields.concat('')" >
```

Comment:
can we avoid calling functions in html? (concat)

Reply:
I thought we want to avoid function calls inside the html templates and we can still use them for attributes but maybe I'm wrong, could you please confirm?

Reply:
Only exception is using them for events like click etc. because we can't avoid binding functions to event and that is fine because then these functions are executed only when event is triggered. 

All other usages like inside {{}}, linking to properties, ngIf etc. are disallowed because they are recalculated with every change detection which affects on performance.

Reply:
@swapnil-verma-gl can you confirm too? You were working on replacing functions in html

Reply:
Okay makes sense, I will move it to .ts part of component

Reply:
@AleksanderSklorz  You are right, function calls in templates should only be reserved for event handlers. All input properties should have a direct variable reference in order to maximise performance. Using functions will cause them to be executed in every change detection cycle. 

Having said that, we did eventually decide that if we come across situations where replacing a function call with a variable proves to be too complex to implement and understand, then in those cases we can leave them in. Angular itself has a few areas where it uses function calls for attributes (Most common that I could find was with forms, where a lot of  functionality is hidden behind getters. For e.g., form.valid).

Reply:
@swapnil-verma-gl thank you for confirmation

Reply:
Fixed